tests/images/e2e/Dockerfile

```diff
@@ -29,5 +30,21 @@ ADD bin/webhook /usr/local/bin/
 ADD bin/blockwriter /usr/local/bin/
 ADD bin/apiserver /usr/local/bin/
 
+RUN mkdir /charts/${RELEASED_VERSION}
+
+RUN curl -L https://github.com/pingcap/tidb-operator/releases/download/${RELEASED_VERSION}/tidb-operator-chart-${RELEASED_VERSION}.tgz \
+    -o tidb-operator-chart-${RELEASED_VERSION}.tgz && \
+    tar -zxvf tidb-operator-chart-${RELEASED_VERSION}.tgz && \
+    mv tidb-operator /charts/${RELEASED_VERSION}/tidb-operator && \
+    rm -rf tidb-operator && \
+    rm tidb-operator-chart-${RELEASED_VERSION}.tgz
+
+RUN curl -L https://github.com/pingcap/tidb-operator/releases/download/${RELEASED_VERSION}/tidb-cluster-chart-${RELEASED_VERSION}.tgz \
+    -o tidb-cluster-chart-${RELEASED_VERSION}.tgz && \
+    tar -zxvf tidb-cluster-chart-${RELEASED_VERSION}.tgz && \
+    mv tidb-cluster /charts/${RELEASED_VERSION}/tidb-cluster && \
+    rm -rf tidb-cluster && \
+    rm tidb-cluster-chart-${RELEASED_VERSION}.tgz
```

Comment:
download these charts in `Upgrading Operator from 1.0.6` test

Reply:
updated, using git clone now.